docs/docs/guides/automate/declarative-automation/automating-asset-checks.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Guide on automating AssetChecks
+sidebar_position: 100
+title: Automating asset checks
+---
+
+While it is most common to use automation conditions to automate the execution of assets, you can also use them to automate the execution of asset checks. This is particularly helpful in cases where you want to automate your data quality checks independently of the assets they are associated with.
+
+## Recommended automation conditions
+
+The core builtin automation conditions are designed to work with asset checks in addition to assets, and can be used and customized in the same way regardless of whether they are applied to an asset or an asset check.
```

Comment:
Since this basically repeats the intro, I think you can remove it, or rewrite the intro to incorporate it.